deploy.env

```diff
@@ -15,9 +15,18 @@ SUPABASE_PASSWORD=root
 REDIS_HOST=redis
 REDIS_PORT=6379
 
+# Minio Host
+MINIO_HOST=minio
+MINIO_PORT=9000
+
+# Gotrue Host
+GOTRUE_HOST=gotrue
+GOTRUE_PORT=9999
+GOTRUE_DATABASE_HOST=postgres
+
 # AppFlowy Cloud
 ## URL that connects to the gotrue docker container
-APPFLOWY_GOTRUE_BASE_URL=http://gotrue:9999
+APPFLOWY_GOTRUE_BASE_URL=http://${GOTRUE_HOST}:${GOTRUE_PORT}
```

Comment:
Since AppFlowy Cloud is installed within the same docker compose network as Gotrue, are there scenarios where we need a different gotrue host?